includes/wikia/models/DesignSystem/WdsImageTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+trait WdsImageTrait {
+	public $image;
+
+	public function setSvgImageData( $key ) {
+		$this->image = $key;
+		$this->{'image-data'} = new WdsSvg( $key );
```

Comment:
implementing jsonSerialize was the previous approach, however after @hakubo showed me this curly-magic-thing it seemed to be more consistent with the rest of the classes